Sources/SourceControl/GitRepository.swift

```diff
@@ -33,9 +33,59 @@ public class GitRepositoryProvider: RepositoryProvider {
 
     /// Reference to process set, if installed.
     private let processSet: ProcessSet?
-
-    public init(processSet: ProcessSet? = nil) {
+    /// The path to the directory where all cached git repositories are stored.
+    private let cachePath: AbsolutePath
+    /// The maximum size of the cache in bytes.
+    private let maxCacheSize: UInt64
```

Comment:
Please separate these properties with a newline for consistency with the rest of the code.

```suggestion
    /// The path to the directory where all cached git repositories are stored.
    private let cachePath: AbsolutePath

    /// The maximum size of the cache in bytes.
    private let maxCacheSize: UInt64
```